src/Microsoft.Health.Dicom.Core/Microsoft.Health.Dicom.Core.csproj

```diff
@@ -14,6 +14,7 @@
     <PackageReference Include="fo-dicom.Imaging.ImageSharp" />
     <PackageReference Include="MediatR" />
     <PackageReference Include="Microsoft.ApplicationInsights" />
+    <PackageReference Include="Microsoft.Data.SqlClient" />
```

Comment:
This is probably a bad sign. This project shouldn't know about SQL.

Reply:
Are there some other ways to do this? For example, is there a way to avoid having each to know about CMK? Incoming requests in ASP.NET Core and outgoing HTTP requests in the Azure SDKs use a pipeline concept where there are discreet steps. Could our health checks leverage a similar pattern?